package.json

```diff
@@ -23,6 +23,7 @@
   "dependencies": {
     "@babel/plugin-transform-modules-commonjs": "^7.18.6",
     "@cloudscape-design/browser-test-tools": "^3.0.2",
+    "@cloudscape-design/component-toolkit": "^1.0.0-beta.53",
```

Comment:
Should this be a dev-dependency?

Reply:
will update - have another PR coming up anyway as this needed some tweaks after further end-to-end testing